KitePrintSDK/src/androidTest/java/ly/kite/BasketOrderTests.java

```diff
@@ -567,6 +579,8 @@ public void testPhotobook1()
     OrderingDataAgent orderingDataAgent = OrderingDataAgent.getInstance( getContext() );
     Product product = new Product( "product_id", "product_code", "Product Name", "Product Type", 0xff000000, UserJourneyType.PHOTOBOOK, 2 );
 
+    KiteSDK.setEncryptionKey("mnk45k3nk5jn4kjn534jkn5kjn34k5j34");
```

Comment:
I think it's around ensuring the tests are more representative of how the developers will actually use the public SDK. I think the more common approach will be setting up encryption via the getInstance() methods :)